packages/calcite-components-react/README.md

```diff
@@ -16,6 +16,8 @@ There are two builds that are provided by the standard `calcite-components` pack
 
 ### Custom Elements build
 
+<!-- TODO: Get suggestions for what this should be replaced with -->
+
```

Comment:
I think we could link to https://developers.arcgis.com/calcite-design-system/get-started/#custom-elements here. Applies to similar build sections.